test/e2e/e2e.go

```diff
@@ -28,6 +28,11 @@ var (
 	ClusterName  string
 	Region       string
 	FileSystemId string
+	// Combined label selectors in the form of key1=value1,key2=value2.
+	// Supplied by users.
+	CombinedEfsDriverLabelSelectors string
```

Comment:
can unexport and move this to e2e_test.go since basically it is an intermediate value that is not useful until parsed